src/main/java/gift/service/TokenService.java

```diff
@@ -45,9 +49,15 @@ public String getMemberIdFromToken(String token) {
     }
 
     public Claims parseToken(String token) {
-        JwtParser parser = (JwtParser) Jwts.parser().setSigningKey(secretKey);
+        JwtParser parser = Jwts.parserBuilder()
+                .setSigningKey(secretKey)
```

Comment:
```suggestion
                .setSigningKey(getSecretKey())
```

로 변경할 필요는 없을까요? 암복호화키가 달라질 것 같은데, 이유가 궁금합니다.

Reply:
이 부분이 문제였네요,, 감사합니다!ㅠ